### DISCUSSION TITLE
High fps in Hitman 2 and Contracts in Whisky

### DISCUSSION THREAD

**Discussion by sejj3u8** [Whisky-App/Whisky#1174] [Q&A]
Good afternoon, please help! I downloaded on macbook pro m1 via Whisky steam, installed old parts of hitman. But already in the menu the game starts to lag because of high fps, in the game itself is the same. Can you please tell me how I can limit or fix fps? Maybe there is such functionality in Whiskey or there is a programme for mac os? Through crossover the same problem

I have already tried to download hitman from torrent, but this problem persists

**Comment by mattmaniak** [comment]
> Good afternoon, please help! I downloaded on macbook pro m1 via Whisky steam, installed old parts of hitman. But already in the menu the game starts to lag because of high fps, in the game itself is the same. Can you please tell me how I can limit or fix fps? Maybe there is such functionality in Whiskey or there is a programme for mac os? Through crossover the same problem
> 
> 
> 
> I have already tried to download hitman from torrent, but this problem persists 

_the game starts to lag because of high fps_ - what do you mean by lagging by high fps?

**Reply by sejj3u8** [reply]
I mean freezes, I guess the game can't run correctly at high fps, so the picture and sound are interrupted
